### ISSUE TITLE
Feature request: Exclusion rule for "Enable Sync" feature

### ISSUE THREAD

**Issue by barnscott** [mozilla/multi-account-containers#1838]
**TLDR:**
Requesting a feature where I can add wildcard rules for names of containers that WONT get sync'd with "Enable Sync" feature. For example, I want to be able to exclude containers that match "tmp*" from syncing.

**Background / reason:**
I use both "Multi-Account-Containers" add-on and "Temporary Containers" add-on. 
When I open a new tab, by default, it opens as a Temporary Container and the cookies will be deleted when the tab is closed. This is true unless the tab is configured to open with a Multi-Account-Container (MAC).

Currently, if "Enable Sync" is turned-on, then all of these temporary containers are also backed up, and it fails to "forget" these temporary containers even after the tab is closed, because Firefox Sync recovers/re-syncs the temporary containers that were intended for termination.

Currently, I have just turned-off the Enable Sync feature, but it would be preferable to still sync the containers **_I had manually created_** with MAC. Since Temporary Containers start with "tmp*", I would like a feature in which I can exclude all containers that begin with "tmp*" from being sync'd.

**Comment by danielfilho** [comment]
Related

- #1748
